util/src/main/scala/testz/futureUtil.scala

```diff
@@ -52,25 +72,25 @@ object futureUtil {
     inner()
   }
 
-  def collectIterator[A](it: Iterator[Future[() => Unit]])(ec: ExecutionContext): Future[() => Unit] = {
+  def collectIterator[A](it: Iterator[Future[A]])(ec: ExecutionContext): Future[List[A]] = {
```

Comment:
doesn't this reverse the order of elements gotten from `it`? could that make a difference to anyone?

Reply:
Hmmm probably. Good to know, deserves a comment.

Reply:
Actually, the synchronous elements were prepended, and the async ones were appended!

Fixed this and made it not reversed.